blog/2023/07/demos-and-tutorials-first-round.md

```diff
@@ -0,0 +1,171 @@
+# Agregore Demos and Tutorials Round One
+
+After working on it for a few months we've finished our first milestone for our [Demos and Tutorials devgrant](/2023/01/demos-and-tutorials-announcement)!
+As outputs from this we have two tutorials, and 4 small apps for folks to try out.
+
+## What we made
+
+### Dirk's DBrowser Development Environment
+
+[@DirkCuys](https://github.com/dirkcuys) put together a step by step guide for bootstrapping a development environment from just the devtools interface in Agregore.
+You can see a live version of the final app [here](/docs/examples/browser-devenv/) or follow step by step from the [tutoial](/docs/tutorials/ipfs-browser-devenv/part-1).
```

Comment:
apple a button to show the editor

Reply:
Maybe the text should read:

> [@DirkCuys](https://github.com/dirkcuys) put together a step by step guide for bootstrapping a development environment from just the devtools interface in Agregore. You can follow step by step from the [tutorial](https://agregore.mauve.moe/docs/tutorials/ipfs-browser-devenv/part-1). This app builds up from uploading your logic via devtools all the way to a fully fledgged GUI which you can use to edit files within your IPFS sites. The final code of the app is here [here](https://agregore.mauve.moe/docs/examples/browser-devenv/).

Still not perfect, since the essence of the tutorial is doing the bootstrap and for the final app to really make sense, we need to link to an IPFS address.